fixtures/ed/ntfs_dst/admin_stations.txt

```diff
@@ -1 +0,0 @@
-admin_id,admin_name,station_id,station_name,stop_id,stop_name
```

Comment:
To me the point is to be able to read that file in the end (well with at least the one line you previously added).
So I think you should keep your test over it also, no?

Reply:
At first, I started to implement my test inside ntfs_dst. Then, I realized that it didn't have `SA` inside `stops.txt`... But we need it to retrieve stop_name.

My future changes will concern `ntfs` and `ntfs_v5` data set, because both are already SA inside stops.txt.
The first test will use stations_id and the other the stop_id.

In a near future, I will open a new PR to erase the retrocompatibility and suppress the test with `station_id`.

About the empty `admin_stations.txt` file inside ntfs_dst, I thought it was useless, but finally I think it is a good way to show, that the file can exist inside the data set, even if it is empty (Good for newbies).
So, I prefer to let it. 
Are you agree with that @pbougue ?

Reply:
Agreed :+1: